docs/source/getting_started/installation.rst

```diff
@@ -108,7 +108,7 @@ You can install ``jupyverse`` with ``pip``:
 
 .. code:: bash
 
-    pip install jupyverse[auth, jupyterlab]
+    pip install jupyverse[auth,jupyterlab]
```

Comment:
I recall the safest syntax is using double quotes (as this will work well on both zsh and Windows):

```suggestion
    pip install "jupyverse[auth,jupyterlab]"
```